kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/listeners/TestListener.kt

```diff
@@ -5,6 +5,9 @@ import io.kotest.core.test.TestCase
 @Deprecated("Renamed to IgnoredSpecListener. Deprecated since 5.0")
 typealias SpecIgnoredListner = IgnoredSpecListener
 
+@Deprecated("Renamed to IgnoredSpecListener. Deprecated since 5.0")
+typealias SpecIgnoredListener = IgnoredSpecListener
+
```

Comment:
Not sure there's a point in adding this typealias, unless `SpecIgnoredListener` existed in Kotest 4?

Reply:
Ah yeah, I forgot to mention this.

The existing deprecated type alias `SpecIgnoredListner` has a typo, `Listner`. The new type alias fixes it, `Listener`. 

Options:

1. keep the misspelling and don't add a fixed version (it's been deprecated for a while, everyone has probably already migrated already)
2. fix the spelling (might break anyone who still uses `SpecIgnoredListner`?)
3. keep the misspelled version and add a fixed version (it might help people migrate?)

I went with 3 because I wanted to have a discussion in the PR. Option 1 is probably best.

Reply:
Had to do some digging.. The previously existing class was typod, so the type alias must also be typod. 

https://github.com/kotest/kotest/blob/v4.6.1/kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/listeners/spec.kt#L49

Reply:
I didn't even consider that the original was misspelled! I've removed my addition and added a comment.